### PR TITLE
Support Ngrok

### PR DESCRIPTION
**Pull request by Anooj-Pai**

### CODE REVIEW

File: cli.js
Comment (curran) [inline cli.js:13]: I believe it should be possible for users to leverage NGrok without VZCode having any direct dependency on it.

File: cli.js
Comment (curran) [inline cli.js:136]: This API key is, I would guess, tied to your personal NGrok account? I would suggest we do not add this to the repo. If there is a strong need to use an API key, it should probably go into an environment variable.

File: localhost.crt
Comment (curran) [inline localhost.crt:1]: Generally the `.key` and `.crt` files should not go in a repository, as these are environment-specific (tied to the particular NGrok account).

File: package.json
Comment (curran) [inline package.json:38]: Nice change! It would be best to make a separate PR so this can get in, independently of the NGrok stuff.

File: package.json
Comment (curran) [inline package.json:57]: I don't think we need this dependency.

File: src/App.jsx
Comment (curran) [inline src/App.jsx:256]: This seems to be a Prettier-related change. I wonder what is going on with this. Maybe if we in introduce a `.prettierrc`, this issue might go away. That would the make VSCode Prettier plugin (which might be the source of this particular change) do the same thing as our `npm run prettier` script.

**Review by curran — CHANGES_REQUESTED** [review]
Let's put this one on hold for a bit.

Generally the `.key` and `.crt` files should not go in a repository.

I believe it should be possible for users to leverage NGrok without VZCode having any direct dependency on it.

Researching a bit,

>You can create basic http-https-tcp tunnel without an [authtoken](https://ngrok.com/docs#authtoken). For custom subdomains and more you should obtain an authtoken by [signing up at ngrok.com](https://ngrok.com/).

I would rather leave it up to users if they want to set up a custom subdomain. I think for most use cases, a basic http-https-tcp tunnel would be sufficient.

**Comment by Anooj-Pai** [comment]
@curran can you call to talk about some of these changes?

**Pull request closed without merging**
